draft-seemann-quic-address-discovery.md

```diff
@@ -138,12 +138,8 @@ receives an OBSERVED_ADDRESS frame.
 An endpoint that negotiated (see {{negotiate-extension}}) this extension and
 offered to provide address observations to the peer MUST send an
 OBSERVED_ADDRESS frame on every new path. This also applies to the path used for
-the QUIC handshake.
-
-The OBSERVED_ADDRESS frame SHOULD be sent as early as possible. However, during
-the handshake an endpoint SHOULD prioritize frames that lead to handshake
-progress (CRYPTO and ACK frames in particular) over sending of the
-OBSERVED_ADDRESS frame.
+the QUIC handshake. The OBSERVED_ADDRESS frame SHOULD be sent as early as
+possible.
 
```

Comment:
Yes. We only send these frames in 1RTT packets, so there is little need to specify timing.